index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&family=Teko:wght@500&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Pink Floyd: the most psychedelic band of all time</h1>
+      <!-- <h1 id="title">Pink Floyd</h1> -->
```

Comment:
It's not good practice to leave commented code, instead of that you can delete it, that makes your code more readable and clear 💯 👍